site/content/docs/main/repository-maintenance.md

```diff
@@ -130,8 +130,23 @@ velero install --default-repo-maintain-frequency <DURATION>
 ```
 For Kopia the default maintenance frequency is 1 hour, and Restic is 7 * 24 hours.
 
+### Full Maintenance Interval customization
+The full maintenance interval defaults to kopia defaults of 24 hours. Velero provide two override options under `fullMaintenanceInterval` configuration.
+- fastGC: 12 hours
```

Comment:
does it makes sense to have `defaultGC` ? I know it's when there are no configuration added, but for usability maybe it makes sense to allow user ensure default is used.

on the second end of the intervals there may be a need to push default to something longer, for the datasets with minimal changes to save on compute costs ?

Reply:
I think we should have default as an explicit option -- it will make changes from eager/fast back to default consistent with the other changes, and it also keeps Velero consistent with 24 hour default even if Kopia changes their  default i the future.

Reply:
Sure.

Reply:
I don't know that we need infrequentGC now -- but we could add it in the future if we get a customer complaining that they don't want to remove deleted blobs within 48 hours.

Reply:
For the name of the default option, I suggest we use `normalGC`. `infrequentGC` sounds more like a value below normal.